nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/FileInfo.java

```diff
@@ -217,6 +217,20 @@ public Builder group(String group) {
         }
     }
 
+    public static String permissionToString(int fileModeOctal) {
```

Comment:
As it is a pure function without side effects, some unit tests could be added easily.